### PR TITLE
Add prop of text-align right to DialogBody

### PR DESCRIPTION
**Pull request by heinburger**
also remove LoadMore component because there is a better solution for this

### CODE REVIEW

File: src/components/DialogBody/DialogBody.js [hunk 1]
```diff
@@ -7,21 +7,27 @@ const defaultStyle = {
...
-  </div>
-)
+  )
+}
 
 DialogBody.propTypes = {
-  backgroundColor: PropTypes.string
+  bgColor: PropTypes.string,
+  style: PropTypes.object,
+  right: PropTypes.bool
```
Comment (menor) [inline src/components/DialogBody/DialogBody.js]: it's better to keep this in alphabetical order

File: src/components/DialogBody/DialogBody.spec.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ import { hexToRgb } from '../../utils/colorUtils'
 import DialogBody from './DialogBody'
 
 const testProps = {
-  backgroundColor: '#111111'
+  bgColor: '#111111'
 }
```
Comment (menor) [inline src/components/DialogBody/DialogBody.spec.js]: I would use `#111`

File: src/components/DialogBody/DialogBody.js (hunk 1 above)
Comment (heinburger) [inline src/components/DialogBody/DialogBody.js]: can we add that to our linter?

Comment (menor) [inline src/components/DialogBody/DialogBody.js]: I don't know

Comment (heinburger) [inline src/components/DialogBody/DialogBody.js]: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/sort-prop-types.md

**Pull request merged**